Kein Ende in Sicht

AurelioSilver opened the pull request:
Noch ca. 200 Datein in data zu überprüfen...

AurelioSilver committed:
Gesetzen -> Gesetze / deine -> dein

AurelioSilver committed:
eine -> ein

AurelioSilver committed:
Schlecht imitiert

AurelioSilver committed:
und -> Und

AurelioSilver committed:
Fürst -> Fürsten

AurelioSilver committed:
zuteil werden -> zuteilwerden

AurelioSilver committed:
dem selben -> demselben

AurelioSilver committed:
näherkommst -> näher kommst

AurelioSilver committed:
+ aber

AurelioSilver committed:
Was für eine Tragödie...

AurelioSilver committed:
wieder -> wider

AurelioSilver committed:
trant -> tarnt

AurelioSilver committed:
aufhaltsam -> unaufhaltsam / das Selbe -> dasselbe / ein -> eine

AurelioSilver committed:
Leid -> leid / Schößlinge -> Schösslinge

AurelioSilver committed:
Altaren -> Altären

Xaratas commented:
Ich hab auch mal ein paar weitere gefunden.

AurelioSilver committed:
Merge remote-tracking branch 'Upstream/master'

AurelioSilver committed:
Gesicht -> Gesichts / Leid -> leid

AurelioSilver committed:
Glauben -> Glaube / Leid -> leid / unterhielten -> unterhalten hatten

AurelioSilver committed:
weiter -> weitere

AurelioSilver committed:
Habicht -> Habichts

AurelioSilver committed:
zuteil werden -> zuteilwerden / dass -> das / fettgefressen -> fett gefressen

AurelioSilver committed:
Komma fehlte

AurelioSilver committed:
zuteil werden -> zuteilwerden

AurelioSilver committed:
Fürst -> Fürsten

AurelioSilver committed:
spricht -> sprichst / glaubt -> glaubst

AurelioSilver committed:
Ort -> Orte

AurelioSilver committed:
Merge remote-tracking branch 'Upstream/master'

AurelioSilver committed:
Gesichts -> Gesicht / mich -> mir

Xaratas commented on data/localized/de_patch/text/conversations/14_burial_isle/14_cv_scathden.stringtable:
```diff
@@ -53,7 +53,7 @@ Er ist in ein Sammelsurium aus zusammengeflickten Fellen und Pelzen von den unte
       <DefaultText>Hiravias tritt vor, streckt die Brust heraus und verschränkt die Arme.
 
-„Ich bin Hiravias, Kind der Fischerkraniche, Adept des Habicht und Efeu-Zirkels, Wanderer von Weretha.“ Seine Stimme ist laut und triumphal, doch kaum hat er seine Vorstellung abgeschlossen, tritt er einen Schritt zurück und flüstert dir zu: „Zu formell?“</DefaultText>
+„Ich bin Hiravias, Kind der Fischerkraniche, Adept des Habichts und Efeu-Zirkels, Wanderer von Weretha.“ Seine Stimme ist laut und triumphal, doch kaum hat er seine Vorstellung abgeschlossen, tritt er einen Schritt zurück und flüstert dir zu: „Zu formell?“</DefaultText>
```
Das s für den Genitiv kommt doch nur hinten an den den Namen der Gruppe sagt mein Grammatikverständnis.

AurelioSilver replied:
Hm, mag sein. Kann es wieder abändern.

AurelioSilver committed:
Habichts -> Habicht

AurelioSilver committed:
Und noch mehr Namen

AurelioSilver committed:
Ein er zuviel / den -> der

AurelioSilver committed:
Achsel -> Achseln / ihre -> ihren

AurelioSilver committed:
Das wrid schon. :) / Symbol -> Symbole

AurelioSilver committed:
ihr -> ihre / diese -> dieses / Glauben -> >Glaube

AurelioSilver committed:
ihrem -> ihren

AurelioSilver committed:
Zwölf -> Zwölfen / würden -> würde / meine -> meinen / Das - Der

AurelioSilver committed:
eine -> einen / Instinkten -> Instinkt / Namen -> Name / ihre -> ich

AurelioSilver committed:
2X gerne ist was viel / Namen -> Name / Es ergibt Sinn / OCR wohl fehlgeschlagen? / Bruder -> Bruders

AurelioSilver committed:
ein Kerze -> eine Kerze

AurelioSilver committed:
begann -> begonnen hatte

AurelioSilver committed:
macht Sinn -> ergibt Sinn

AurelioSilver committed:
Plateau -> Plateaus

AurelioSilver committed:
Willen -> willen / sie -> Sie

AurelioSilver committed:
wir -> wird / wievie -> wie viel

AurelioSilver committed:
liegen gelassen -> liegen lassen

AurelioSilver committed:
Beine -> Beinen

AurelioSilver committed:
werden -> werde / Jedesmal -> Jedes Mal / Augen -> Auge / Gesichts -> Gesicht

AurelioSilver committed:
Gesichts -> Gesicht / + du

Xaratas commented:
Alles aus dem Language Tool heraus? Mit allen Regeln oder nur mit einem Subset?

AurelioSilver commented:
Ja, mit dem tool. Und allen Regeln... ist was mühselig. Alles was mir merkwürdig vorkommt, schaue ich mir dann an und schaue auch im Duden nach,

AurelioSilver committed:
+ Kornstab

AurelioSilver committed:
Kleine Aktualisierung der Namen fürs Language tool

AurelioSilver committed:
dieses -> dieser / wenigsten -> wenigstens
ein Kunst-> eine Kunst / ersuchte -> ersucht hatte
umformuliert

AurelioSilver committed:
Fixes from Kornstab
https://forums.obsidian.net/topic/81258-translation-errors-missing-translations-german/?p=1803071

Xaratas commented on data/localized/de_patch/text/conversations/companions/companion_cv_hiravias_quest.stringtable:
```diff
@@ -39,7 +39,7 @@ Seine Stimme wird leiser und er erblasst. „Bin ich die geringere Seele? War me
       <DefaultText>„Ich weiß es nicht, aber es klingt danach, als würden sie nur versuchen, ihre Morde an Leuten zu rechtfertigen.“
 
-„Komm schon, lass uns mal sehen, was auf den anderen Tafel geschrieben steht.“</DefaultText>
+„Komm schon, lass uns mal sehen, was auf dem anderen Tafel geschrieben steht.“</DefaultText>
```
Das sollte doch sicher ein „der“ werden.

AurelioSilver replied:
Grr. ja.

Xaratas commented on data/localized/de_patch/text/conversations/companions/companion_cv_hiravias_quest.stringtable:
```diff
@@ -189,7 +189,7 @@ Er ballt seine Fäuste und lässt einen langen Seufzer los. „Ja, das relativie
     <Entry>
       <ID>34</ID>
-      <DefaultText>„Ich bezweifle, dass irgendein Teil dieses Erkenntnisse die Meinungen meines Heimatstammes ändern werden, und nachdem ich da draußen einige andere druidische Methoden kennengelernt habe, werde ich wohl lieber ein Herumtreiber bleiben.“</DefaultText>
+      <DefaultText>„Ich bezweifle, dass irgendein Teil dieser Erkenntnisse die Meinungen meines Heimatstammes ändern werden, und nachdem ich da draußen einige andere druidische Methoden kennengelernt habe, werde ich wohl lieber ein Herumtreiber bleiben.“</DefaultText>
```
Das „werden“ müsste ein „wird“ sein. Und Meinungen im Plural wenn von der ganzen Gruppe gesprochen wird ist auch sehr gestreckt. Würde da eine Meinung draus machen.

AurelioSilver committed:
dem -> der

Da hat die Autokorrektur dazwischengefunkt.

AurelioSilver committed:
Korrekturvorschlag Xaratas

AurelioSilver committed:
mir -> mit

AurelioSilver committed:
auch -> auf

AurelioSilver committed:
zu -> du / . -> , / diesen -> dieser

AurelioSilver committed:
neues -> Neues / es ergibt Sinn

AurelioSilver committed:
wie > wir / deinen -> deiner / ähnliches -> Ähnliches
Zähne -> Zähnen / vier Mal -> viermal / es ergibt Sinn

AurelioSilver committed:
Willen -> willen / Namen -> Name

AurelioSilver committed:
Geflüstere -> Geflüster

AurelioSilver committed:
Grad -> Grat / Einmischungen -> Einmischung / Es egibt Sinn.

AurelioSilver committed:
mir -> mit

AurelioSilver committed:
Äh... ja...

Xaratas commented:
Ich nehm das jetzt mal mit rein, damit ich nicht Gefahr laufe das alles doppelt zu finden.
